.github/workflows/test.yml

```diff
@@ -16,20 +16,15 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-24.04, macos-14, windows-2022]
-        feature-args: ['', '-Funstable-mobile-app']
         include:
           - os: ubuntu-24.04
             display-os: Linux
           - os: macos-14
             display-os: macOS
           - os: windows-2022
             display-os: Windows
-          - feature-args: ''
```

Comment:
Removed this too as it was originally added in https://github.com/getsentry/sentry-cli/pull/2582, but the empty args could be useful for having around if future flags are added so `${{ matrix.feature-suffix }}` won't need to be added below again. Will leave to you @szokeasaurusrex if that's alright.